docs/governance/tokenomics.md

```diff
@@ -2,200 +2,178 @@
 
-A delegator is someone who delegates any amount of tokens to one or more validators.
+Validators and Delegators earn **Block rewards** for keeping the blockchain operational and secure.
+**Block rewards** consist of **Staking rewards** and **Transaction fees**.
 
-Tokens delegated to a validator, including its self-delegated tokens, are taken out of the delegator’s account and kept in a pool.
+**Tx fees** should be the main source of revenue for **Validators** and **Delegators**. However, due to the expectation of low early engagement numbers on the **Cudos network**, additional rewards are provided to users. 
 
-The validator and its delegators share the rewards based on the amounts they have contributed to this pool.
+:::tip Validator commission
 
-A validator can additionally collect a commission fee from its delegators.
+Validators need to use the `--commission` flag in `cudos-noded CLI` in order to withdraw their commission.
+:::
 
-At start, a validator defines a commission rate, a max change rate, and a max commission rate.
+### Block Rewards 
 
-Then it can change its commission rate once every day by at most its max change rate as long as the commision rate doesn’t become greater than the max commission rate.
+The more tokens delegated to a **Validator node**, the higher the likelihood of it being chosen to propose and sign a block. 
 
-Validators are ranked based on the amount of tokens that have been delegated to them, including their self-delegations.
+* **Block rewards** are calculated after a block is issued. 
 
-At each block, the top 100 validators who are not jailed are said to be bonded.
+:::info Block rewards
 
-If, for any reason, a validator goes outside the bonded validators set, it enters an unbonding period.
-A validator that has failed to get itself back into the bonded validators set within 21 days is said to be unbonded.
-The number 100 can be updated via a simple governance proposal. – Need testing and verification.
+*Block rewards* =  *staking rewards* + *tx fees* 
+:::
 
-**Only bonded validators can propose or sign blocks.
-Only bonded validators can earn block rewards.
-Only bonded validators’ tokens are subject to slashing.**
+### Staking rewards 
 
-Although the actual act of slashing might occur during the unbonding period.
+Staking rewards occur in a **single pool** **Tx fees** should be the main source of revenue for **validators** and **delegators**. However, due to the expectation of low early engagement numbers on the **Cudos network**, additional rewards are provided to users. and are distributed as follows:  
 
-## Redelegation 
+1. **Bonded validators** (including the **Block proposer**) are assigned rewards proportional to the amount they have staked relative to the entire network. 
```

Comment:
I suggest we replace "Bonded validators" with "Bonded token holders" and remove the bit inside the paranthesis.

Reply:
Done